frontend/src/pulse/components/PulseEdit.jsx

```diff
@@ -41,15 +42,23 @@ export default class PulseEdit extends Component {
         this.props.dispatch(fetchCards());
         this.props.dispatch(fetchUsers());
         this.props.dispatch(fetchPulseFormInput());
+
+        MetabaseAnalytics.trackEvent((this.props.pulseId) ? "PulseEdit" : "PulseCreate", "Start");
     }
 
     async save() {
         await this.props.dispatch(saveEditingPulse());
+
+        MetabaseAnalytics.trackEvent((this.props.pulseId) ? "PulseEdit" : "PulseCreate", "Complete", this.props.pulse.cards.length);
```

Comment:
What does the event payload (specifically the # cards) end up looking like in GA?